kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/clusternetworkaddressconfigprovider/PortPerBrokerClusterNetworkAddressConfigProvider.java

```diff
@@ -50,6 +50,7 @@ public static class PortPerBrokerClusterNetworkAddressConfigProviderConfig {
         @JsonIgnore
         private final RangeAwarePortPerNodeClusterNetworkAddressConfigProvider.RangeAwarePortPerNodeClusterNetworkAddressConfigProviderConfig rangeAwareConfig;
         private final HostPort bootstrapAddress;
+
```

Comment:
Does this PR effectively deprecated `PortPerBroker` in favour of `RangeAwarePortPerNode` ? As I don't see any way to configure `PortPerBroker` via this PR.


 I don't have any objections to doing so but I can't remember @robobario logic for preserving `PortPerBroker` when he added `RangeAware` but I think it was discussed.

Reply:
> Does this PR effectively deprecated PortPerBroker in favour of RangeAwarePortPerNode ? 

In effect, yes.

If I'm not mistaken, everything you can achieve with PortPerBroker you can achieve with RangeAwarePortPerNode (with an appropriately defined range).  Indeed, PPB is actually already implemented with RAPPN (see io.kroxylicious.proxy.internal.clusternetworkaddressconfigprovider.PortPerBrokerClusterNetworkAddressConfigProvider.PortPerBrokerClusterNetworkAddressConfigProviderConfig#PortPerBrokerClusterNetworkAddressConfigProviderConfig)

Reply:
In that case should we open a PR deprecating it and raise a card to remove it in N+2 ?

Reply:
we decided not to deprecate at the time since it was a bit experimental

https://github.com/kroxylicious/kroxylicious/pull/1220#discussion_r1611353900

Reply:
I think all the ClusterNetworkAddressConfigProviders implementations can be removed at N+2 as they will be gone from the config model.  There needs to be _something_ to replace them that will bridge between SniHostIdentifiesNodeIdentificationStrategy and PortIdentifiesNodeIdentificationStrategy and the EndpointRegistry (currently the NodeIdentificationStrategy lean on the ClusterNetworkAddressConfigProviders).  
I think we can defer this work.

Reply:
We should add the depreciation tags though so our n+2 clock starts ticking.